Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -367,6 +367,16 @@ void WebFullScreenManager::didEnterFullScreen()
         return;
     }
 
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+    const FloatSize maxViewportSize(10000, 10000);
```

Comment:
Nit - I think we generally prefer `constexpr` when possible.

```suggestion
    static constexpr auto maxViewportSize = FloatSize { 10000, 10000 };
```